src/popup/router/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <Modal class="confirm-tx-sign-modal" :close="false">
```

Comment:
```suggestion
  <Modal class="confirm-tx-sign-modal">
```

Reply:
Actually I think this should stay, because this modal should not have close button.

Reply:
`false` is a default value of `close` prop in Modal, so it can be omitted

Reply:
I forgot to make `close` prop in Modal defaults to` true`, because in most of the cases our modals need to have close button.

Reply:
I like to have Bool attributes in Vue `false` by default because `:attr-name="false"` looks worse than `not-attr-name`